fission-cli/library/Fission/CLI/Release.hs

```diff
@@ -0,0 +1,63 @@
+module Fission.CLI.Release (checkLatestRelease) where
+
+import           Data.Versions
+import qualified Data.Yaml                               as YAML
+import qualified System.Console.ANSI                    as ANSI
+
+import           Fission.Prelude
+
+import           Fission.Error.NotFound.Types
+
+import           Fission.CLI.Environment                as Env
+import qualified Fission.CLI.Meta                       as Meta
+import           Fission.CLI.Display.Text
+import qualified Fission.CLI.Display.Error       as CLI.Error
+
+import qualified Fission.Internal.UTF8                  as UTF8
+
+import           GitHub (github')
+import qualified GitHub
+import           GitHub.Data.Releases                   as Releases
+
+checkLatestRelease ::
+  ( MonadLogger      m
+  , MonadIO          m
+  , MonadTime        m
+  , MonadCleanup     m
+  , MonadEnvironment m
+  , m `Raises` YAML.ParseException
+  , m `Raises` NotFound FilePath
+  , Show (OpenUnion (Errors m))
+  )
+  => m ()
+checkLatestRelease = do
+  attempt Env.get >>= \case
+    Left  err -> CLI.Error.put err "Unable to parse config"
```

Comment:
Super minor style OPTIONAL thing that I find useful but is nonstandard: If any branch in a `case` drops to the next line, have them all drop to the next line and place a blank line between. I find it makes visually parsing each branch a bit easier, but also maybe just me 🤷‍♀️